partner_contact_show_ref/models/res_partner.py

```diff
@@ -1,14 +1,17 @@
 # Copyright 2025 Alberto Martínez <alberto.martinez@sygel.es>
 # License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
 
-from odoo import models
+from odoo import fields, models
 
 
 class Partner(models.Model):
     _inherit = "res.partner"
 
+    own_ref_in_name = fields.Boolean()
```

Comment:
Why by partner? Wouldn't this be a company or global configuration in the system?